main.c

```diff
@@ -513,6 +513,9 @@ static void *tcmur_cmdproc_thread(void *arg)
 			else
 				ret = tcmur_generic_handle_cmd(dev, cmd);
 
+			if (ret == TCMU_NOT_HANDLED)
+				tcmu_warn("not handled command %x\n", cmd->cdb[0]);
+
```

Comment:
If we add it here, I think we want to remove it from all other places. For example, handle_generic_cmd and fbo_handle_cmd.